datasets/simulated.py

```diff
@@ -12,8 +12,9 @@ class Dataset(BaseDataset):
     # the cross product for each key in the dictionary.
     parameters = {
         'n_samples, n_features': [
-            (100, 5_000),
-            (100, 10_000)
+            (100, 10_000),
+            (500, 600),
+            (10_000, 100)
```

Comment:
I'd remove this one. It's not a default Lasso setting in my opinion

Reply:
Interesting, why do you not think it's a lasso setting? Because lars/homotopy would just make much more sense here? What if we increased n? Something like 10000 x 500 ?

Reply:
Now we can set them how we want with the CLI, but I think it's still nice to have sensible default.
In order to keep default benchs fast, I'd suggest 100 x 1000, 500 x 500, 1000 x 100, what do you think ?

Reply:
Absolutely, but maybe 100x100 instead of 500x500? 500x500 is going to be much harder for most solvers than 100 x 1000. I also think we should do something about the number of true signals so make sure it's not larger than min(n, p).